inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/matrix_nms.cpp

```diff
@@ -8,14 +8,22 @@
 #include "single_layer_tests/matrix_nms.hpp"
 #include "common_test_utils/test_constants.hpp"
 
-using namespace LayerTestsDefinitions;
+using namespace ov::test::subgraph;
 using namespace InferenceEngine;
 using namespace ngraph;
+const std::vector<ShapeParams> inStaticShapeParams = {
+    // dynamic shape, {{batch, box, 4}, {batch, class, box}}, out if static shape
+    ShapeParams{{}, {{{3, 100, 4}, {3,   1, 100}}}, true},
+    ShapeParams{{}, {{{1, 10,  4}, {1, 100, 10 }}}, false}
+};
 
-const std::vector<InputShapeParams> inShapeParams = {
-    InputShapeParams{3, 100, 5},
-    InputShapeParams{1, 10, 50},
-    InputShapeParams{2, 50, 50}
+const std::vector<ShapeParams> inDynamicShapeParams = {
+    ShapeParams{{{ngraph::Dimension::dynamic(), 100, 4}, {ngraph::Dimension::dynamic(), 5, 100}},
```

Comment:
please add test cases where all inputs shape is undefended (`ngraph::Dimension::dynamic(`), all input shapes have defined lower and upper bounds

Reply:
Done.